### PR TITLE
[MRG] Fix compute source psd pick ori

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: mne/minimum_norm/tests/test_time_frequency.py [hunk 1]
```diff
@@ -97,19 +98,25 @@ def test_source_psd():
...
-                             method="dSPM", tmin=tmin, tmax=tmax,
-                             fmin=fmin, fmax=fmax, pick_ori="normal",
-                             n_fft=n_fft, label=label, overlap=0.1)
-    assert_true(stc.times[0] >= fmin * 1e-3)
-    assert_true(stc.times[-1] <= fmax * 1e-3)
-    # Time max at line frequency (60 Hz in US)
-    assert_true(59e-3 <= stc.times[np.argmax(np.sum(stc.data, axis=0))] <=
-                61e-3)
+
+    assert_true(inverse_operator['source_ori'] == FIFF.FIFFV_MNE_FREE_ORI)
```
Comment (Eric89GXL) [inline mne/minimum_norm/tests/test_time_frequency.py]: `assert_equal` is preferred in these instances (more informative when it fails)

**Comment by agramfort** [comment]
your wish is my command @Eric89GXL

**Comment by Eric89GXL** [comment]
Thanks @annapasca and @agramfort

**Comment by annapasca** [comment]
thanks to you!

**Pull request merged**